metaflow/metaflow_config.py

```diff
@@ -312,6 +312,9 @@
 ARGO_EVENTS_INTERNAL_WEBHOOK_URL = from_conf(
     "ARGO_EVENTS_INTERNAL_WEBHOOK_URL", ARGO_EVENTS_WEBHOOK_URL
 )
+ARGO_EVENTS_SENSOR_NAMESPACE = from_conf(
```

Comment:
Is the use case you have such that all deployed sensors would end up in the same namespace, or are there a bunch of different ones?

I'm wondering whether to add the new namespace as part of the CLI configure command as well, or is it expected that users set the relevant env var when deploying their flows?

Reply:
Yes, our use case is that all sensors live in the same namespace and that we would always set the env var along with other metaflow config env vars when running on kubernetes.  We generally put these config things into a .env file for each repo that uses metaflow so there would be no need for a CLI option in our case.  

Generally, we run workflows in one of two kubernetes namespaces and all of our metaflow + argo related services (including the argo-events eventbus) live in a third namespace.  While we have two namespaces now this could grow and I didn't want to put an eventbus and webhook into each namespace.

Reply:
I just pushed a commit to use this namespace in the delete command.